components/automate-deployment/pkg/backup/spec.go

```diff
@@ -416,6 +416,10 @@ func DefaultSpecs(serviceNames []string) []Spec {
 				},
 			},
 		},
+		{
+			Name:          "automate-cs-ocid",
+			WriteMetadata: false,
```

Comment:
need clarity on this. can discuss.

Reply:
Need clarity from automate team.

Reply:
@YashviJain01 @daveaugustus Can you please explain what this boolean does - `WriteMetadata`?